blt/src/Blt/Plugin/Commands/HsCommands.php

```diff
@@ -17,16 +17,19 @@ class HsCommands extends BltTasks {
 
   use HsCommandTrait;
 
+  const UPDATE_PARALLEL_PROCESSES = 'UPDATE_PARALLEL_PROCESSES';
+
   /**
    * After code deployed, update all sites on the stack.
    *
    * @command humsci:post-code-deploy
    *
    * @aliases humsci:post-code-update
    */
-  public function postCodeDeployUpdate() {
+  public function postCodeDeployUpdate($target_env, $deployed_tag) {
     $sites = $this->getConfigValue('multisites');
-    $parallel_executions = 10;
+    $parallel_executions = (int) getenv(self::UPDATE_PARALLEL_PROCESSES) ?: 10;
```

Comment:
I don't 100% understand the need to create the `UPDATE_PARALLEL_PROCESSES` constant here to call out the name of the environment variable.

Reply:
- waves hand over it -
To make it look more fancy.

Reply:
![fancy-cat](https://github.com/SU-HSDO/suhumsci/assets/6943710/e8cacf18-97c1-469f-97e0-32120f41b4f2)